app/grandchallenge/algorithms/models.py

```diff
@@ -923,3 +930,12 @@ def delete(self, *args, **kwargs):
 
     class Meta(RequestBase.Meta):
         unique_together = (("algorithm", "user"),)
+
+
+class OptionalProtocolsAlgorithm(models.Model):
```

Comment:
Lets have consistency of naming with the rest of the codebase. `Protocols` should be `HangingProtocol`, non-plural.

Same for the other apps.